exporter/exporter.go

```diff
@@ -74,6 +72,8 @@ type Exporter struct {
 	api     *client.UNMSAPI
 	metrics map[string]*prom.Desc
 
+	withInterfaces bool
```

Comment:
i don't see this field being read anywhere?
is it supposed to turn off reporting for all interfaces or just the WAN interface? what would be the use of either?

Reply:
This sets the `withInterfaces` parameter for the `/devices` endpoint. Previously, this was hard-coded to use `exporter.defaultWithInterfaces`, and can now be configured with `(*Exporter).EnableWANMetrics(bool)`. (I've renamed the field to `wanMetrics` for clarification).